OracleApplicationExpress/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Use Oracle 19c image as the base
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# Set environment variables
+ENV PATH=$ORACLE_HOME/bin:$PATH
+
+USER root
+# Install Java 11 OpenJDK
+RUN yum install -y java-11-openjdk-devel
```

Comment:
Please add `&& rm -rf /var/cache/yum` to this directive to reduce the size of the layer.